react/src/AddDatasets/components/DataEntryToolbar.tsx

```diff
@@ -107,7 +108,11 @@ export default function DataEntryToolbar(props: {
                     </IconButton>
                 </Tooltip>
                 <Tooltip title="Go to MinIO">
-                    <Link href={process.env.REACT_APP_MINIO_URL} target="_blank" rel="noreferrer">
+                    <Link
+                        href={getActiveMinioEndpoint() || process.env.REACT_APP_MINIO_URL}
```

Comment:
Since all use cases have the ` || process.env.REACT_APP_MINIO_URL` pattern, you can just include that in the function instead.